tensorrt_llm/_torch/attention_backend/sparse/rocketkv.py

```
@@ -0,0 +1,1202 @@
+        Uses the manager's public ``rewind_kv_cache`` (=
+        ``self.impl.rewind_kv_cache``) to shrink the cache to
+        prompt_budget+1.
+        """
+        # Skip terminated mid-prefill
+        try:
+            from tensorrt_llm._torch.pyexecutor.llm_request import (
+                LlmRequestState)
+            if request.state == LlmRequestState.GENERATION_COMPLETE:
+                return
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)